src/v2/Apps/Order/Components/SavedAddressItem.tsx

```diff
@@ -105,6 +106,8 @@ const EditButton = styled(Text)`
 `
 
 const StyledPill = styled(Pill)`
-  background-color: ${color("black10")} !important;
-  pointer-events: none;
+  background-color: ${themeGet("colors.black10")};
+  &:hover {
+    background-color: ${themeGet("colors.black10")};
+  }
```

Comment:
I'm just wondering why there's this one-off Pill variation in the first place? Is this something that should be elevated to a proper variant in Pill in the design system?

Reply:
I think the better solution is to update the `Pill` component in `Palette`. In the near future, I will try to figure out how to do it.

Reply:
I know for sure that something is wrong with the definition of the background of the component. This is complicated by the fact that the background changes in the parent component (`RadioGroup`) on the `hover`

Reply:
Honestly: Design System atoms like this aren't meant to have custom styling, that's really it. There's nothing broken or wrong, exactly. It may just be the case that design has asked for a snowflake here and you don't actually want a `Pill`. The valid variants can be seen here: https://palette-storybook.artsy.net/?path=/story/components-pill--variants

Reply:
I agree with @dzucconi - this is technically a fine solution, but ideologically we're constrained by what the design system supports. If a designer is trying to pass off a design with a color (etc) that isn't in Palette then the design system wins over the designer, fortunately. 

I would just use the regular pill and not worry about this, even if it feels a little weird. If it's a big deal we can have a formal discussion. In the meantime tho, we shouldn't be doing 1-off things.